guides/v2.2/config-guide/cli/config-cli-subcommands-index.md

```diff
@@ -126,11 +126,12 @@ Reindexing all indexers can take a long time for stores with large numbers of pr
 
 ### Reindex in parallel mode {#config-cli-subcommands-index-reindex-parallel}
 
-As of Magento 2.2.5, indexers are scoped and multi-threaded to support reindexing in parallel mode. This feature reduces processing time. It parallelizes by the indexer's dimension and executes across multiple threads.
+As of Magento 2.2.6, indexers are scoped and multi-threaded to support reindexing in parallel mode. This feature reduces processing time. It parallelizes by the indexer's dimension and executes across multiple threads.
 
 The following indexes can be run in parallel mode:
 - Catalog Search Fulltext can be paralleled by store views.
 - Category Product can be paralleled by store views.
```

Comment:
"Catalog Search Fulltext can be paralleled by store views." should be changed to "Catalog Search Fulltext is parallelized by store views."
"Category Product can be paralleled by store views" should be changed to "Category Product is parallelized by store views."